Fix test_more_recent_version_pending_rejection_as_well intermittent test failure

```
src/olympia/reviewers/tests/test_commands.py:922: in test_more_recent_version_pending_rejection_as_well
    assert more_recent_version.version not in message.body
E   AssertionError
```

`more_recent_version` has no fixed version number so the assertion will intermittently fail.

--- a/src/olympia/reviewers/tests/test_commands.py
+++ b/src/olympia/reviewers/tests/test_commands.py
@@ -907,7 +907,7 @@ class TestSendPendingRejectionLastWarningNotification(TestCase):
             ActivityLog.create(
                 amo.LOG.REJECT_VERSION_DELAYED,
                 addon, version, details={'comments': 'fôo'}, user=self.user)
-        more_recent_version = version_factory(addon=addon)
+        more_recent_version = version_factory(addon=addon, version='43.0')
         VersionReviewerFlags.objects.create(
             version=more_recent_version,
             pending_rejection=datetime.now() + timedelta(days=3))

